code/obj/mining.dm

```diff
@@ -779,7 +779,7 @@
 			return 1
 
 		var/rangecheck = 0
-		if (issilicon(usr))
+		if (isAI(usr))
```

Comment:
This stops borgs from using it

Reply:
isAI() checks for silicon or AIeye
`#define isAI(x) (istype(x, /mob/living/silicon/ai) || istype (x, /mob/living/intangible/aieye))`

Reply:
It checks for silicon/**ai** not silicon.